### PR TITLE
Fix test broken by change to stops live data

### PR DESCRIPTION
**Pull request by phildarnowsky**
No ticket. Another live data fix nailed down with ExVCR.

---

Before getting review, please check the following:

* [x] Does frontend functionality render and work correctly in IE?
* [x] Have we load-tested any new pages or internal API endpoints that will receive significant traffic?
* [x] Are interactive elements accessible to screen readers?
* [x] Have you checked for tech debt you can address in the area you're working in?
* [x] If this change involves routes, does it work correctly with pertinent "unusual" routes such as the combined Green Line, Silver Line, Foxboro commuter rail, and single-direction bus routes like the 170?
* [x] Are the changes organized into self-contained commits with descriptive and well-formatted commit messages?

### CODE REVIEW

File: apps/stops/fixture/vcr_cassettes/elixir_stops_apitest__parses_parent_id_and_child_ids.json [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+      "type": "ok"
+    }
+  },
+  {
+    "request": {
+      "body": "",
+      "headers": {
+        "accept-encoding": "gzip",
+        "accept": "application/vnd.api+json",
+        "x-api-key": "1c05fc0118144698888cc5a1b08d3a56"
```
Comment (paulswartz) [inline apps/stops/fixture/vcr_cassettes/elixir_stops_apitest__parses_parent_id_and_child_ids.json:48]: This shouldn't be checked in. I'm going to disable this key and I'll share a different one with you.

Are there other API keys in the other cassettes?

Comment (phildarnowsky) [reply apps/stops/fixture/vcr_cassettes/elixir_stops_apitest__parses_parent_id_and_child_ids.json:48]: @paulswartz it's supposed to filter that header out, and in fact in other places in this same cassette it does so--this is very odd. Possibly a bug in ExVCR?

**Comment by phildarnowsky** [comment]
@paulswartz corrected the pertinent lines manually. I'm concerned about ExVCR not consistently blanking out that key, will open a ticket to look into that.

**Comment by alicia-mbta** [comment]
A couple notes:
- From my investigation so far, it seems that ExVCR expects a 'default' configuration, so I got the filtering of the headers to work by adding `config :exvcr, filter_request_headers: ["x-api-key"]` in file `dotcom/config/config.exs`. My understanding is that this `config.exs` file is for the whole application so I'm not sure if this change is an issue or not.

- This change successfully filtered the "x-api-key" values in the cassettes but I still can't get the vcr test in file `dotcom/apps/site/test/site_web/controllers/fare_controller_test.exs` to pass. I'm going to continue digging.

**Pull request merged**